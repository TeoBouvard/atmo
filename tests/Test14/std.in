1
0
0
50000

2017-01-03
5