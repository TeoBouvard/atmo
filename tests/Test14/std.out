Importation des capteurs ... 1 capteurs importÃ©s
Importation des mesures ... 8 mesures importÃ©es

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter


Latitude : Longitude : Rayon : Date de dÃ©but au format YYYY-MM-DD : Date de fin au format YYYY-MM-DD : 
Analyse en cours ... 

[1m[37mCapteur(s) selectionnÃ©(s) : 

Sensor nÂ°0 | Distance au centre : 1568.52km

Nombre de mesures sur cette pÃ©riode de temps : 4

[1m[36mMoyenne 03 : 0 Âµg/m3 | Sous indice 03 : 1
Moyenne NO2 : 0 Âµg/m3 | Sous indice NO2 : 1
Moyenne SO2 : 0 Âµg/m3 | Sous indice SO2 : 1
Moyenne PM10 : 0 Âµg/m3 | Sous indice PM10 : 1

[1m[31mIndice ATMO : 1[0m

Analyse terminÃ©e.

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter
